ecs/daemon/outputs/s3/task.conf

```diff
@@ -0,0 +1,45 @@
+[FILTER]
+    Name              ecs
+    Match             app.var.lib.docker.containers.*
+    ECS_Tag_Prefix    app.var.lib.docker.containers.
+    ADD               container    $ECSContainerName
+    ADD               ecs_task_id  $TaskID
+    ADD               ecs_family   $TaskDefinitionFamily
+
+[FILTER]
+    Name               aws
+    Match              app.var.lib.docker.containers.*
+    imds_version       v1
+    az                 false
+    ec2_instance_id    true
+
+
+[FILTER]
+    Name                  rewrite_tag
+    Match                 app.var.lib.docker.containers.*
+    Rule                  $container ^[\S]+$  task.$ecs_family.$ec2_instance_id.$ecs_task_id.$container false
+    Emitter_Name          re_emitted_task
+    Emitter_Storage.type  filesystem
+    Emitter_Mem_Buf_Limit ${MEM_BUF_LIMIT_REWRITE_TAG_TASK}
+
+
+[OUTPUT]
+    Name                         s3
+    Match                        task.*
+    bucket                       ${BUCKET_TASK}
+    region                       ${REGION_TASK}
+    upload_timeout               2m
+    total_file_size              10M
+    s3_key_format                /aws/ecs/${CLUSTER_NAME}/$TAG[0]/$TAG[1]/$TAG[2]/$TAG[3]/$TAG[4]/%Y/%m/%d/%H/%M/%S
+    s3_key_format_tag_delimiters .
+    store_dir                    /var/fluent-bit/s3-store/task
+
+[OUTPUT]
+    Name                         s3
+    Match                        app.*
+    bucket                       ${BUCKET_TASK}
+    region                       ${REGION_TASK}
+    upload_timeout               2m
+    total_file_size              10M
+    s3_key_format                /aws/ecs/${CLUSTER_NAME}/task/failed-to-retrieve-metadata-for/$TAG/%Y/%m/%d/%H/%M/%S
```

Comment:
If I have two rewrite_tag filters, one for the fallback logs, I could still add instance ID probably to them, to make it still somewhat possible to figure where the failed logs came from.

Reply:
Need to update timestamp here as well